akka-http2-support/src/test/scala/akka/http/impl/engine/http2/WithPriorKnowledgeSpec.scala

```diff
@@ -16,10 +17,11 @@ import akka.util.ByteString
 import scala.concurrent.Future
 
 class WithPriorKnowledgeSpec extends AkkaSpec("""
-    akka.loglevel = warning
+    akka.loglevel = debug
     akka.loggers = ["akka.testkit.TestEventListener"]
```

Comment:
It also need the `SilenceAll...` listener, otherwise, stuff will still be printed.